File: API/Entities/SkillTag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Skillustrator.Api.Entities
{
    public class SkillTag : EntityBase
    {
        public int SkillId { get; set; }

        public Skill Skill { get; set; }

        public int TagId { get; set; }

        public Tag Tag { get; set; }
    }
}

File: UI/src/app/models/skill-tag.ts
import { Skill } from './skill';
import { Tag } from './tag';

export class SkillTag {
  id: number;
  skillId: number;
  skill: Skill;
  tagId: number;
  tag: Tag;
  
  constructor(id: number, skillId: number, skill: Skill, tagId: number, tag: Tag) {
    this.id = id;
    this.skillId = skillId;
    this.skill = skill;
    this.tagId = tagId;
    this.tag = tag;
  }
}
